extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -568,9 +590,17 @@ export class JupyterServerInstallation implements IJupyterServerInstallation {
 		}
 	}
 
-	public async getInstalledPipPackages(): Promise<PythonPkgDetails[]> {
+	public async getInstalledPipPackages(pythonExePath?: string): Promise<PythonPkgDetails[]> {
+		if (pythonExePath) {
+			if (!fs.existsSync(pythonExePath)) {
```

Comment:
Do you want to wrap this in a try/catch in case there's some other problem accessing the path?